osquery/events/linux/syslog.cpp

```diff
@@ -55,12 +55,18 @@ const char* kTimeFormat = "%Y-%m-%dT%H:%M:%S";
 const std::vector<std::string> kCsvFields = {
     "time", "host", "severity", "facility", "tag", "message"};
 const size_t kErrorThreshold = 10;
+static const size_t MAX_LINE_LEN = 16384;
```

Comment:
Please follow the conventions that there are in this file about global constants. 
```suggestion
const size_t kMaxLineLen = 16384;
```